### ISSUE TITLE
A11Y: User cannot unfocus the editor

### ISSUE THREAD

**Issue by matyax** [grafana/grafana#57247]
**Issue**

In explore, when navigating with the keyboard, once the cursor focuses in the editor, you cannot escape. If you press tab, tabs will be added as content of the editor.

**Expected behavior**

Tab should focus the next element after the editor.

**Comment by matyax** [comment]
I received feedback from in the accessibility ambassadors meeting that the expected behavior here is to press `escape` to exit the editor and continue navigating using `tab`.
